webui/react/src/components/BreadcrumbBar.tsx

```diff
@@ -131,6 +131,18 @@ const BreadcrumbBar: React.FC<Props> = ({
     setWorkspace(workspaceIn);
   }, [workspaceIn]);
 
+  // cleanup
+  useEffect(() => {
+    return () => {
+      stopPolling();
+
+      setWorkspace(undefined);
+      setProject(undefined);
+      setExperiment(undefined);
+      setTrial(undefined);
```

Comment:
not sure these are actually meaningful or not, but it removes warning and I supposed there's no bad effects from them